vendor/gridjson.js

```diff
@@ -0,0 +1,545 @@
+
+function _classCallCheck(instance, Constructor) { if (!(instance instanceof Constructor)) { throw new TypeError("Cannot call a class as a function"); } }
+
+var GoogleMap = function () {
+    function GoogleMap(nativeMap) {
+        _classCallCheck(this, GoogleMap);
+
+        this._map = nativeMap;
+    }
+
+    /**
+     * Return the pixel coordinates of an event at a certain zoom level.
+     */
+
+
+    _createClass(GoogleMap, [{
+        key: "project",
+        value: function project(event) {
+            var latLng = event.latLng;
+            var point = this._map.getProjection().fromLatLngToPoint(latLng);
+            var scale = Math.pow(2, this.getZoom());
+            return {
+                x: Math.floor(scale * point.x),
+                y: Math.floor(scale * point.y)
+            };
+        }
+    }, {
+        key: "getZoom",
+        value: function getZoom() {
+            return this._map.getZoom();
+        }
+    }, {
+        key: "on",
+        value: function on(event, callback) {
+            this._map.addListener(event, callback);
+        }
+    }]);
+
+    return GoogleMap;
+}();
+
+module.exports = GoogleMap;
+
+/***/ }),
+/* 3 */
+/***/ (function(module, exports, __webpack_require__) {
+
+"use strict";
+
+
+var _createClass = function () { function defineProperties(target, props) { for (var i = 0; i < props.length; i++) { var descriptor = props[i]; descriptor.enumerable = descriptor.enumerable || false; descriptor.configurable = true; if ("value" in descriptor) descriptor.writable = true; Object.defineProperty(target, descriptor.key, descriptor); } } return function (Constructor, protoProps, staticProps) { if (protoProps) defineProperties(Constructor.prototype, protoProps); if (staticProps) defineProperties(Constructor, staticProps); return Constructor; }; }();
+
+function _classCallCheck(instance, Constructor) { if (!(instance instanceof Constructor)) { throw new TypeError("Cannot call a class as a function"); } }
+
+var LeafletMap = function () {
+    function LeafletMap(nativeMap) {
+        _classCallCheck(this, LeafletMap);
+
+        this._map = nativeMap;
+    }
+
+    // LatLong to pixel coords
+
+
+    _createClass(LeafletMap, [{
+        key: "project",
+        value: function project(event) {
+            var latlng = event.latlng;
+            return this._map.project(latlng, this._map.getZoom()).floor();
+        }
+    }, {
+        key: "getZoom",
+        value: function getZoom() {
+            return this._map.getZoom();
+        }
+    }, {
+        key: "on",
+        value: function on(event, callback) {
+            this._map.on(event, callback);
+        }
+    }]);
+
+    return LeafletMap;
+}();
+
+module.exports = LeafletMap;
+
+/***/ })
+/******/ ]);
+});
+//# sourceMappingURL=gridjson.bundle.js.map
```

Comment:
Is this code in another repo?

This is an already bundled code. Where we can find the original?